src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -348,8 +353,9 @@ private void ensureApkOutputDirectoriesExist() throws IOException {
     private boolean processPrecompiledClasses(String androidJar) throws IOException, InterruptedException {
         String androidCodeLocation = "/native/android/dalvik";
 
+        copyOtherDalvikClasses();
         if (projectConfiguration.isUsePrecompiledCode()) {
-            copyPrecompiledClasses(androidCodeLocation + "/precompiled/class/");
+            copyPrecompiledClasses(androidCodeLocation + Constants.DALVIK_PRECOMPILED_CLASS);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Constants.DALVIK_PRECOMPILED_CLASS' due to existing static import 'com.gluonhq.substrate.Constants.DALVIK_PRECOMPILED_CLASS'](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=5183430)